fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/IndexKeyValueToPartialRecord.java

```diff
@@ -93,6 +95,67 @@ void copy(@Nonnull Descriptors.Descriptor recordDescriptor, @Nonnull Message.Bui
                   @Nonnull IndexEntry kv);
     }
 
+    /**
+     * The copier to populate the lucene auto complete suggestion as a value for the field where it is indexed from.
```

Comment:
umm I guess we can let the client to control if they want the suggestion to be with highlighting or not (through index options for now). So I guess there is no need to tell if the suggestion result is with highlight or not in the response. And in the future we probably can support clients to select this in each request rather than configuring it in schema.